services/net/transcription/TranscriptionManager.cs

```diff
@@ -152,6 +152,7 @@ private async Task HandleMessageAsync(ConsumeResult<string, TranscriptRequest> r
         var content = await _api.FindContentByIdAsync(result.Message.Value.ContentId);
         if (content != null)
         {
+            // TODO: Handle multi-threading so that more than one transcription can be performed at a time.
```

Comment:
Currently the design of this service results in only a single transcription being processed.  Generally speaking it's a 1:1 process, for every minute of speech it takes a minute to process.  This means the Transcription Service locks up for a long time doing nothing.  However, it's not so easily solved either.  If we multi-thread we need to be able to limit the number, handle errors, and safely shutdown.